File: src/test/java/model/ArtistTest.java
package model;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class ArtistTest {
    private Artist artist;
    private Artist artist1;

    @BeforeEach
    void setUp() {
        artist = new Artist(2L, "Kygo");
        artist1 = new Artist("Kygo");
    }

    @Test
    void getName() {
        assertEquals(artist.getName(), "Kygo");
        assertEquals(artist1.getName(), "Kygo");
    }

    @Test
    void setName() {
        artist.setName("Twenty One Pilots");
        artist1.setName("Twenty One Pilots");
        assertEquals(artist.getName(), "Twenty One Pilots");
        assertEquals(artist1.getName(), "Twenty One Pilots");
    }
}

File: Tests/model/ArtistTest.cs
using MusicFestivalMPP.model;
using NUnit.Framework;

namespace Tests.model;

public class ArtistTest
{
    private Artist _artist;
    private Artist _artist0; 
    
    [SetUp]
    public void Setup()
    {
        _artist = new Artist(2, "stefan");
        _artist0 = new Artist("mara");
    }

    [Test]
    public void GetName()
    {
        Assert.AreEqual(_artist.Name, "stefan");
        Assert.AreEqual(_artist0.Name, "mara");
    }
    
    [Test]
    public void SetName()
    {
        _artist.Name = "Stefan";
        Assert.AreEqual(_artist.Name, "Stefan");
        Assert.AreEqual(_artist0.Name, "mara");
    }
}